config/sync/views.view.driving_instructors.yml

```diff
@@ -427,3 +440,891 @@ display:
         - 'user.node_grants:view'
       tags:
         - 'config:search_api.index.contacts'
+  rest_export_1:
```

Comment:
Would be ideal to have a more descriptive display id. The defaults views produces can be ambiguous and easily missed in the config UI